localstorage.js

```diff
@@ -0,0 +1,22 @@
+const storage = document.querySelector('.container-form');
+const email = document.getElementById('email');
+const userName = document.getElementById('name');
+const message = document.getElementById('msg');
+const object = {};
+
+window.addEventListener('DOMContentLoaded', () => {
+  const userInput = JSON.parse(localStorage.getItem('data'));
+  if (userInput !== null) {
+    userName.value = userInput.nameID;
+    email.value = userInput.mail;
+    message.value = userInput.msg;
+  }
+});
+
+// Add to localStorage
+storage.addEventListener('input', () => {
+  object.nameID = userName.value;
+  object.mail = email.value;
+  object.msg = message.value;
+  localStorage.setItem('data', JSON.stringify(object));
+});
```

Comment:
- [ ] Great job adding the localstorage.js file 💯, but you forgot to add the reference of the file into your `index.html` 🤪. Kindly, add this script, and your project will work. 🍨
- [ ] Also, please check the placeholder id(msg) input, because the data from it, is not sent to the localstorage. ⚡